schema_registry/client/urls.py

```diff
@@ -7,7 +7,10 @@ class UrlManager:
     def __init__(self, base_url: str, paths: list) -> None:
         parsed_url = urllib.parse.urlparse(base_url)
 
-        assert parsed_url.scheme, f"The url does not have a schema, add one. For example http://{base_url}"
+        assert parsed_url.scheme in (
+            "http",
+            "https",
+        ), f"The url {base_url} has invalid schema. Use http or https. For example http://{base_url}"
```

Comment:
After [bpo-27657](https://docs.python.org/3/whatsnew/changelog.html) was fixed in [python 3.9 alpha 1](https://docs.python.org/3/whatsnew/changelog.html#python-3-9-0-alpha-1) `urllib.parse.urlparse('localhost:8080')` leads to the scheme="localhost" unlike scheme=None in python<3.9

Reply:
I think this is fine!!